hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -133,6 +134,16 @@ def hashes(key=None):
     return results if results else {}
 
 
+@app.get("/hash_count")
```

Comment:
Sorry, late to the party, @jeberl, but the route should be `/hash-count` if left this way, but probably a better route would be `/hash/count` since `/hash` is the main "concept". @BarrettOlson since you might be the consumer of this at some point, would you fix at that time?

Reply:
@jeberl and @BarrettOlson I know I missed making this comment when the PR was open. Let me know if making this comment here at this time is confusing / not a best practice. If there's another way to communicate this better, let me know.